praetorian_cli/sdk/entities/assets.py

```
@@ -76,27 +77,44 @@ def delete(self, key):
-        if prefix_filter:
-            dns_prefix = f'group:{prefix_filter}'
-        if asset_type == '':
-            asset_type = Kind.ASSET.value
-        return self.api.search.by_term(dns_prefix, asset_type, offset, pages)
+
+        if asset_type in KIND_TO_LABEL:
+            asset_type = KIND_TO_LABEL[asset_type]
+        elif not asset_type:
+            asset_type = Node.Label.ASSET
+        else:
+            raise ValueError(f'Invalid asset type: {asset_type}')
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)